Makefile

```diff
@@ -0,0 +1,95 @@
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
+all: build
+
+verify: fmt vet lint
+
+# Run tests
+test: generate verify manifests unit
+
+unit:
+	hack/unit-tests.sh
```

Comment:
Broken target

Reply:
I know it's broken, my plan is to make a follow-up PR with missing tasks. I didn't want to add too many changes here.

Reply:
This pr contains 1kk LOC btw...

Reply:
It's all vendor directory :D